UnityProject/Assets/Scripts/Transform/CustomNetTransform.cs

```diff
@@ -542,6 +545,22 @@ private void AdminRespawn()
 	{
 		PlayerManager.PlayerScript.playerNetworkActions.CmdAdminRespawn(gameObject);
 	}
+
+	// Checks if the object is occupiable and update occupant position if it's occupied (ex: a chair)
+	private void UpdateOccupant()
+	{
+		// Since we are at the transform level, we go up one level to get the parent
```

Comment:
Shouldn't need to go up a level, GetComponent should be fine. CustomNetTransform lives on the same level as all the other components